semantic-analysis/semantic-analysis.cabal

```diff
@@ -44,24 +44,24 @@ library
   import: common
   hs-source-dirs: src
   exposed-modules:
+    Analysis.Analysis.Concrete
+    Analysis.Analysis.Exception
+    Analysis.Analysis.Typecheck
```

Comment:
Moving these really just gets them out of the way so they don't look like the mechanisms underlying interpretation. E.g. if we added an interval analysis and also used intervals here and there, we'd have conflicts.